indico/modules/events/logs/controllers.py

```diff
@@ -29,17 +31,20 @@ class RHEventLogs(RHManageEventBase):
     """Shows the modification/action log for the event"""
 
     def _process(self):
+        data_query = {k[len('filter.'):]: int(v) if v.isdigit() else v
```

Comment:
And maybe using `filter.` here will generate confusion, since we're calling "filters" something else?

Reply:
I had suggested on IRC to rename our filters since we call those realms everywhere else. But with your `meta` suggestion I think `meta` would fit nicely here as well.